pipeline_dp/accumulator.py

```diff
@@ -33,6 +33,13 @@ def create_accumulator_params(
         accumulator_params.append(
             AccumulatorParams(accumulator_type=CountAccumulator,
                               constructor_params=CountParams()))
+    if pipeline_dp.Metrics.SUM in aggregation_params.metrics:
```

Comment:
could you please add testing of Sum parameters [here](https://github.com/OpenMined/PipelineDP/blob/4049178721a5ef9e013dfce398356b9d3388dfad/tests/accumulator_test.py#L252)

Reply:
Done, thanks! 👍